libraries/_resoruce_helpers.rb

```diff
@@ -107,6 +107,42 @@ def cp_r(src, dst)
       ::FileUtils.chown_R(node['knotx']['user'], node['knotx']['group'], dst)
     end
 
+    def get_top_level_distro_directory(tmp_dir_path)
```

Comment:
I'd rename that to `get_top_level_distro_directories` as there may be more than 1

Reply:
I will change `select` to `find`.